modules/magniteAnalyticsAdapter.js

```diff
@@ -123,11 +123,15 @@ const addEventToQueue = (event, auctionId, eventName) => {
   }
 }
 
-const sendEvent = payload => {
+const sendEvent = (payload, isUnload) => {
```

Comment:
I'd add a default value for isUnload so it's clear why you're not passing it: 

```
const sendEvent = (payload, isUnload = false) => { 
```

this is already implicit, it's just cosmetic and to make it more explicit... 
```

Reply:
it shall be done